app/controllers/application_controller.rb

```diff
@@ -35,7 +35,7 @@ def with_current_request
         else
           ActsAsTenant.without_tenant { yield }
         end
-      rescue TopologicalInventory::Api::NoTenantError, KeyError
+      rescue TopologicalInventory::Api::NoTenantError, KeyError, ManageIQ::API::Common::IdentityError
```

Comment:
Do we need our own NoTenantError anymore?  It looks like `current.user.tenant` might blow up before we get our own keyerror.

Reply:
I think you are right. That error was raised before we started auto creating the tenant. Let me verify that.

Reply:
@agrare I pushed another commit to remove `NoTenantError`

Reply:
:+1: